en_US/messages.json

```
@@ -134,5 +134,9 @@
           "example" : "100"
         }
       }
+    },
+    "tlsIncompatibility": {
+      "message": "Translations only works on secure (HTTPS) pages.",
```

Sorry, I only now noticed how strange this looks. It should either say "Firefox Translations", or "Translation" (as the act of translating). The first reaction as a user is that the English is broken.

```suggestion
      "message": "Firefox Translations only works on secure (HTTPS) pages.",
```